spec/features/admin/user_orders/show_spec.rb

```diff
@@ -50,5 +50,63 @@
         expect(page).to have_content("Subtotal: $#{item_order_2.subtotal}")
       end
     end
+
+    describe 'then click a link to visit a user\'s order show page' do
+      it 'it displays a link to cancel the order' do
+        user_1 = User.create(name: 'User 1', address: '123 Main', city: 'Denver', state: 'CO', zip: 80_233, email: 'user_1@user.com', password: 'secure', role: 0)
+        dog_shop = Merchant.create(name: "Brian's Dog Shop", address: '125 Doggo St.', city: 'Denver', state: 'CO', zip: 80_210)
+        bike_shop = Merchant.create(name: "Brian's Bike Shop", address: '125 Bike St.', city: 'Denver', state: 'CO', zip: 80_210)
+        tire = bike_shop.items.create(name: 'Gatorskins', description: "They'll never pop!", price: 100, image: 'https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588', inventory: 12)
+        pull_toy = dog_shop.items.create(name: 'Pull Toy', description: 'Great pull toy!', price: 10, image: 'http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg', inventory: 32)
+        order_1 = user_1.orders.create(name: 'User 1', address: '123 Main', city: 'Denver', state: 'CO', zip: 80_233, status: 0)
+        item_order_1 = order_1.item_orders.create(order_id: order_1.id, item_id: tire.id, quantity: 2, price: 15, merchant_id: bike_shop.id, status: 1)
+        item_order_2 = order_1.item_orders.create(order_id: order_1.id, item_id: pull_toy.id, quantity: 1, price: 100, merchant_id: dog_shop.id, status: 0)
+
+        site_admin = User.create(name: 'Site Admin', address: '123 First', city: 'Denver', state: 'CO', zip: 80_233, email: 'site_admin@user.com', password: 'secure', role: 3)
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(site_admin)
+
+        visit "/admin/users/#{user_1.id}/orders/#{order_1.id}"
```

Comment:
Did the path helper not function as expected here?

Reply:
@freeheeling I didn't try as I was trying to remain consistent on the spec. I also think it helps to give clear documentation instead of using the path helper.